docs/setup/xml-config.md

```diff
@@ -14,6 +14,13 @@ export RMW_IMPLEMENTATION=name-of-the-rmw
 ```
 Currently, the only supported RMW implementations are: `rmw_cyclonedds_cpp` and `rmw_fastrtps_cpp`.  See [here](https://docs.ros.org/en/rolling/Concepts/About-Different-Middleware-Vendors.html) for more info on the different RMW vendors.
 
+If you have already set your default RMW implementation via the ROS2 setup process and would like to change your default RMW, use 
```

Comment:
I think that many users may not already have a line in their bashrc that sets the default RMW.
This isn't a required step and it's just mentioned in our setup instructions (people may have followed different instructions to setup ros).

 - I would change this line to say something like:
 "You can set a default rmw adding this line in that file.. You may already have it if you followed our setup (with link to the setup), in that case you can change the name of the RMW...

- don't assume that people have a bashrc file. The `nano` command will create one for you if it's not already there, however bashrc is used only by bash shells and different ones may have a different file that serves that purpose (e.g. zshrc for zsh shells)

 - after modifying that file, you need either to start a new shell or call 
```
source ~/.bashrc
```